### DISCUSSION TITLE
Upload collection should allow providing a description/hint message for the upload field

### DISCUSSION THREAD

**Discussion by finkinfridom** [payloadcms/payload#1998] [Feature Requests & Ideas]
We use several `upload`- based collections, one of which has a long list of available `mimeTypes`.
Unfortunately, the users cannot know in advance which are the allowed types and we cannot provide a meaningful description of them.
Could be an option to have an `hintText` or `description` into `admin.upload` object?

Thanks

**Comment by DanRibbens** [comment]
Not specific to uploads, the admin.description on a collection should do it:  https://payloadcms.com/docs/configuration/collections#admin-options

I believe this is shown everywhere you will need it.

**Reply by finkinfridom** [reply]
Well, the "admin.description" should be used to describe the collection itself not how the uploads for that collection should work.

Let say I have a "media" collection that is also able to ingest items from an external source, then the "admin.description" field could be too loose to describe a single specific field (or could lead to semantically partially incorrect description).
What do you think?

**Comment by revnelson** [comment]
After running into this same issue, the workaround I've come up with is to add a UI field with a simple component:


```javascript
{
			type: "ui",
			name: "uploadNotes",
			label: "Upload Notes",
			admin: {
				components: {
					Field: () =>
						TextComponent({
							text: "This note will be visible when uploading a file."
						})
				}
			}
		}
```

Component:
```typescript
import * as React from "react"

const TextComponent = ({ text }: { text: string }) => {
	return <div>{text}</div>
}

export default TextComponent
```

Unfortunately this is also visible any time you open the document. It'd be far more ideal to be able to add a `description` (or `hint` as OP mentioned) for the file upload field itself (where it says to drag or select a file).
